api/src/controllers/publications.js

```diff
@@ -185,13 +180,11 @@ async function getGoogleScholarPublications(req, res) {
 
     await browser.close();
 
-    await Promise.all(links.map((x) => scrapeGoogleScholar(x))).then((pub) =>
-      publications.push(...pub)
-    );
+    await Promise.all(links.map((x) => scrapeGoogleScholar(x))).then((pub) => publications.push(...pub));
```

Comment:
Not an issue, but I'm surprised you're able to modify arrays when they're initialized to const (L155)? I always thought you couldn't.

Reply:
yeah I believe you cannot re-assign them, hence `const`, but you are able to mutate the contents.